engine/class_modules/paladin/sc_paladin.cpp

```diff
@@ -173,6 +172,7 @@ struct avenging_wrath_t : public paladin_spell_t
       background = true;
 
     harmful = false;
+    use_off_gcd = true;
```

Comment:
Actions that are off the global cooldown can be used before triggering the global cooldown with another action at the same timestamp. Setting ``use_off_gcd=true`` will make the action usable *at any time* including when the global cooldown isn't ready yet. This is not always necessary and can be set on each apl line individually for the same result as setting it in the constructor.
Unless you want to apply use_off_gcd to every Avenging Wrath line that will be added to the APL, I'd recommend not setting it in the constructor.